tendrl/commons/__init__.py

```diff
@@ -23,37 +23,14 @@ def __init__(self, ns_name="tendrl", ns_src="tendrl.commons"):
             the condition when the node_agent has not been started and name
             spaces are being created.
         '''
-        try:
-            Event(
-                Message(
-                    priority="info",
-                    publisher=NS.publisher_id,
-                    payload={"message": "Creating namespace.%s from source %s"
-                                        % (ns_name, ns_src)
-                             }
-                )
-            )
-        except KeyError:
-            sys.stdout.write("Creating namespace.%s from source %s\n" % (
-                ns_name, ns_src))
-
+        Logger.log("info",NS.get("publisher_id", None),message="Creating namespace.%s from source %s" % (ns_name, ns_src))
         self.ns_name = ns_name
         self.ns_src = ns_src
 
         self._create_ns()
 
         self.current_ns = self._get_ns()
-        try:
-            Event(
-                Message(
-                    priority="info",
-                    publisher=NS.publisher_id,
-                    payload={"message": "namespace.%s created!" % self.ns_name
-                             }
-                )
-            )
-        except KeyError:
-            sys.stdout.write("namespace.%s created!\n" % self.ns_name)
+	Logger.log("info",NS.get("publisher_id", None),message="namespace.%s created!" % self.ns_name)
```

Comment:
Thanks for the points, I have made the required changes,  The solution is pretty naive, I am working on making it more generic